Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    public class Lesson3
+    {
+        public static void Demo ()
+        {
+            string userFirstName = UserName(message: "Enter your first name: ");
+            string userSurname = UserName(message: "Enter your surname: ");
+            int userAge = UserAge(message: "Enter your age: ");
+            float userHeight = UserHeightWeight(message: "Enter your height in cm: ");
+            float userWeight = UserHeightWeight(message: "Enter your weight in kg: ");
+            float userBMI = CalculateBmi(userWeight, userHeight);
+            Console.WriteLine(userFirstName + " " + userSurname + " is " + userAge + " years old, his weight is " + userWeight + " kg and his height is " + userHeight + " cm.");
+            Console.WriteLine("BMI = " + userBMI);
+        }
+
+        public static float CalculateBmi(float weight, float height)
+        {
+            double bmi = weight / Math.Pow(height, 2);
+            return Convert.ToSingle(bmi);
+        }
+
+        public static float UserHeightWeight(string message)
+        {
+            Console.Write(message);
+            var heightOrWeight = Console.ReadLine();
```

Comment:
If you run into variable names like this your alarm bell should ring. You're probably doing something in a rather unhealthy fashion.